src/indigenousLayers.js

```diff
@@ -148,6 +138,7 @@ L.LayerGroup.IndigenousLandsTerritoriesLayer = L.LayerGroup.extend(
     }
 );
 
-L.layerGroup.indigenousLandsTerritoriesLayer = function (options) {
-    return new L.LayerGroup.IndigenousLandsTerritoriesLayer(options);
+L.layerGroup.indigenousLayers = function (url,name,options) {
```

Comment:
Removing URL from here as well

Reply:
In this section of the code i basically pass url and name as a parameter both of which are being used.
```
L.layerGroup.indigenousLayers = function (url,name,options) {
        
    return new L.LayerGroup.IndigenousLayers(url,name,options);
};
```
url is the unique url for the 3 layers
name as in territories, languages, treaties